### PR TITLE
add code of conduct text and allow <br /> characters

### PR DESCRIPTION
**Pull request by saturnflyer**
This text has been added to encourage participation of attendees from all walks of life and to ensure
that all attendees are aware of they way they are expected to treat each other. Wroclove.rb is a public
conference and public behavior requires etiquitte which is respectful of every individual.

### CODE REVIEW

File: public/css/style-narrower.css [hunk 1]
```diff
@@ -17,7 +17,7 @@
 /* Section/Article */
 
 	header br {
-		display: none;
+/*    display: none;*/
```
Comment (saturnflyer) [inline public/css/style-narrower.css:20]: I'm happy to change this according to your design, but I commented it out to allow for the line breaks in the text to visually separate the content.

**Comment by mrnugget** [comment]
Merging this would be great! :)

But I think the repo is out of sync with the actual site. The site already has the schedule on it.

File: public/index.html [hunk 2]
```diff
@@ -150,14 +150,32 @@
...
-                question.<br/>Additionally, there will be a special team of <b>student volunteers</b> who will be
-                visible during the conference (If you want to join the team, please let us know). Student volunteers are
-                here to help find your way around and resolve any problem. It is needless to say that this conference is
+                of all the people inside like monitoring and special security people.<br/> It is needless to say that this conference is
                 a place for good people, only. Good people respect each other, are nice to each other, smile, and make
                 everyone (without exceptions) feel comfortable.</p>
+            <body>
+            <h3>Conference Code of Conduct</h3>
+
+            <p>All attendees, speakers, sponsors and volunteers at our conference are required to agree with the following code of conduct. Organisers will enforce this code throughout the event. We are expecting cooperation from all participants to help ensuring a safe environment for everybody.</p>
```
Comment (panSarin) [inline public/index.html]: Good point, but ii`m not sure if it is okey to add this point, when ppl bought tickets.

File: public/index.html [hunk 3]
```diff
@@ -150,14 +150,32 @@
...
+
+            <p><strong>Need Help?</strong> As part of our goal, we’re dedicated to react to all situations that we’ll be notified, ideally directly to us.There will be a <b>special phone number</b>, you can call at any time to let us know about any problem or ask any question.<br/> You have our contact details in the emails we've sent.<br />There will be a special phone number, you can call at any time to let us know about any problem or ask any question. Additionally, there will be a special team of <b>student volunteers</b> who will be visible during the conference (If you want to join the team, please let us know). Student volunteers are here to help find your way around and resolve any problem.</p>
```
Comment (panSarin) [inline public/index.html:160]: +1

**Comment by saturnflyer** [comment]
Don't see the _Schedule_ in the repo @mrnugget so something might be out of sync

**Comment by fidel** [comment]
@saturnflyer, @mrnugget we'll make it synced asap.

**Pull request merged**